### PR TITLE
v1.11.0 - local legends icons

### PR DESCRIPTION
**Pull request by sonnyprince**
- Addition of local legends icons

Documentation PR:
https://github.com/justeat/global-component-library/pull/59

### CODE REVIEW

File: src/scss/partials/_general.scss [hunk 1]
```diff
@@ -90,3 +90,13 @@
     height: 20px;
     width: 20px;
 }
+
+.c-icon--localLegend {
+    height: 32px;
+    width: 87px;
+}
+
+.c-icon--localLegendCondensed {
```
Comment (DamianMullins) [inline src/scss/partials/_general.scss]: Maybe `c-icon--localLegend--condensed` to follow the format used elsewhere in Fozzie?

Comment (sonnyprince) [reply src/scss/partials/_general.scss]: done :)

File: src/img/icons/general/localLegendCondensed.svg [hunk 2]
```diff
@@ -0,0 +1 @@
..."o"></g><polygon id="Shape" fill="#1DB4E9" fill-rule="nonzero" mask="url(#mask-16)" points="12.4693333 0.681411765 -12.5944 13.2225882 -13.5277333 1.70541176"></polygon></g></g><g id="Group" transform="translate(5.600000, 15.583059)"><g id="Clipped"><mask id="mask-18" fill="white"><use xlink:href="#path-17"></use></mask><g id="q"></g><polygon id="Shape" fill="#D2D700" fill-rule="nonzero" mask="url(#mask-18)" points="15.9264 18.8235294 0.877333333 17.1764706 11.536 0.681411765"></polygon></g></g><g id="Group" transform="translate(11.200000, 18.406588)"><g id="Clipped"><mask id="mask-20" fill="white"><use xlink:href="#path-19"></use></mask><g id="s"></g><polygon id="Shape" fill="#93C01F" fill-rule="nonzero" mask="url(#mask-20)" points="24.4561333 13.7308235 0.472266667 13.7308235 5.936 -2.14211765"></polygon></g></g><g id="Group" transform="translate(16.800000, 24.053647)"><g id="Clipped"><mask id="mask-22" fill="white"><use xlink:href="#path-21"></use></mask><g id="u"></g><polygon id="Shape" fill="#00A65D" fill-rule="nonzero" mask="url(#mask-22)" points="27.7386667 8.08376471 18.8561333 8.08376471 0.336 -7.78917647 27.7386667 -4.33505882"></polygon></g></g></g></g></g></g></g></svg>
```
Comment (ashleynolan) [inline src/img/icons/general/localLegendCondensed.svg]: Worth updating the filename to `localLegend-condensed.svg` too

File: src/img/icons/general/localLegend-condensed.svg [hunk 3]
```diff
@@ -0,0 +1 @@
..."o"></g><polygon id="Shape" fill="#1DB4E9" fill-rule="nonzero" mask="url(#mask-16)" points="12.4693333 0.681411765 -12.5944 13.2225882 -13.5277333 1.70541176"></polygon></g></g><g id="Group" transform="translate(5.600000, 15.583059)"><g id="Clipped"><mask id="mask-18" fill="white"><use xlink:href="#path-17"></use></mask><g id="q"></g><polygon id="Shape" fill="#D2D700" fill-rule="nonzero" mask="url(#mask-18)" points="15.9264 18.8235294 0.877333333 17.1764706 11.536 0.681411765"></polygon></g></g><g id="Group" transform="translate(11.200000, 18.406588)"><g id="Clipped"><mask id="mask-20" fill="white"><use xlink:href="#path-19"></use></mask><g id="s"></g><polygon id="Shape" fill="#93C01F" fill-rule="nonzero" mask="url(#mask-20)" points="24.4561333 13.7308235 0.472266667 13.7308235 5.936 -2.14211765"></polygon></g></g><g id="Group" transform="translate(16.800000, 24.053647)"><g id="Clipped"><mask id="mask-22" fill="white"><use xlink:href="#path-21"></use></mask><g id="u"></g><polygon id="Shape" fill="#00A65D" fill-rule="nonzero" mask="url(#mask-22)" points="27.7386667 8.08376471 18.8561333 8.08376471 0.336 -7.78917647 27.7386667 -4.33505882"></polygon></g></g></g></g></g></g></g></svg>
```
Comment (markedup) [inline src/img/icons/general/localLegend-condensed.svg]: Version statement here but not in the non-condensed version--should this be removed here, or added to the non-condensed icon as well?

Comment (sonnyprince) [reply src/img/icons/general/localLegend-condensed.svg]: can / will remove :)

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v1.11.0
+------------------------------
+*September 14, 2018*
```
Comment (80sgreek) [inline CHANGELOG.md]: date update

File: src/scss/partials/_general.scss [hunk 5]
```diff
@@ -90,3 +90,13 @@
     height: 20px;
     width: 20px;
 }
+
+.c-icon--localLegend {
+    height: 32px;
+    width: 87px;
+}
+
+.c-icon--localLegend--condensed {
```
Comment (80sgreek) [inline src/scss/partials/_general.scss]: The double modifier always seems weird to me but I've seen it done before so i guess that's what we do ¯\_(ツ)_/¯

**Pull request merged**